### PR TITLE
IOD-125 Create DeploymentConfig object

### PR DESCRIPTION
**Pull request by akroon3r**
# Description

This PR includes the following proposed change(s):

IOD-125

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by SDToews — COMMENTED** [review]
Should the sha be hard coded on lines 47 and 65?

**Comment by akroon3r** [comment]
Those are just log inputs from the last time the image trigger was changed. They're auto-generated lines inserted by openshift.

File: openshift/templates/citz-imdb-slam-app/dc.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+- description: License Plate
+  displayName: License Plate
+  name: LICENSE_PLATE
+  required: true
+  value: ca94a8
+- description: Environment
+  displayName: Environment
+  name: ENVIRONMENT
+  required: true
+  value: dev
```
Comment (akroon3r) [inline openshift/templates/citz-imdb-slam-app/dc.yaml:91]:
```suggestion
  value: dev
  
```

File: openshift/templates/citz-imdb-slam-app/dc.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
```
Comment (akroon3r) [inline openshift/templates/citz-imdb-slam-app/dc.yaml:92]:
```suggestion

```

Comment (akroon3r) [reply openshift/templates/citz-imdb-slam-app/dc.yaml:92]: typo

File: openshift/templates/citz-imdb-slam-app/dc.yaml (hunk 1 above)
Comment (akroon3r) [inline openshift/templates/citz-imdb-slam-app/dc.yaml:91]:
```suggestion
  value: dev

```

Comment (akroon3r) [reply openshift/templates/citz-imdb-slam-app/dc.yaml:91]: clrf

**Review by SDToews — APPROVED** [review]

**Pull request merged**